realm/realm-jni/src/io_realm_internal_tableview.cpp

```diff
@@ -1015,3 +1015,17 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_TableView_nativeSync(
     } CATCH_STD()
     return 0;
 }
+
+JNIEXPORT jlong JNICALL Java_io_realm_internal_TableView_nativeFindBySourceNdx
+        (JNIEnv *env, jobject, jlong nativeViewPtr, jlong rowIndex)
```

Comment:
`rowIndex` -> `sourceIndex` to match the name in core. `rowIndex` would be confusing.